prep-javascript-conditionals/main.js

```diff
@@ -0,0 +1,78 @@
+const age = 30;
+function isAdult (age) {
+ let lessthan18;
+  if (age<18) {
+  lessthan18 = "true";
```

Comment:
when you want to return a boolean value, you want to use `true` or `false` not `"true"` or `"false"`

the reason is that `true` is a boolean value and `"true"` is a string value.